app/components/auto_tag_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AutoTagComponent < ViewComponent::Base
+  def initialize(text:)
+    colours = {
+      "to do": "yellow",
+      "cannot start yet": "grey",
+      "setup complete": "green",
+      "done": "green",
+    }
```

Comment:
Can we move that to a constant outside `initialize` method? We don't need to make a copy for each instance of the component :)

Reply:
gooood point, i put it in there to debug and then forgot about it 😆